### DISCUSSION TITLE
Canonical-Ubuntu-20.04 AArch64

### DISCUSSION THREAD

**Discussion by magnored** [amidaware/tacticalrmm#1091] [General]
I have an instance on oracle cloud and I can't install the tactical on an aarch64 system. it is not compatible ?

**Comment by dinger1986** [comment]
The simple answer is no it's not compatible but it can be made to work.

You would need to change the install script to use the correct dependencies for your architecture and also compile nats-api.

**Comment by magnored** [comment]
ok...thanks for the feedback!
